.github/workflows/shared_deploy_cloudhub_1_0.yml

```diff
@@ -88,20 +89,12 @@ jobs:
         with:
           ref: ${{ inputs.checkout_ref }}
 
-      # Remove when switching to the new Exchange - BUSINESS_GROUP_ID_GLOBAL_2 completely for all ENV (DEV, STAGING, PROD)
-      - name: Modify groupId in pom.xml
-        shell: bash
-        run: |
-          perl -0777 -pe 's/<groupId>[0-9a-fA-F\-]+<\/groupId>/<groupId>${{ secrets.BUSINESS_GROUP_ID }}<\/groupId>/g' -i.bak pom.xml
-          cat pom.xml
-
-      # Remove when switching to the new Exchange - BUSINESS_GROUP_ID_GLOBAL_2 completely for all ENV (DEV, STAGING, PROD)
-      - name: Modify groupId in Main API Interface
-        if: ${{ inputs.main_api_interface_path }}
+      - name: Comment out Log4j2 File Appender
+        if: inputs.disable_log4j2_file_appender == 'true'
         shell: bash
         run: |
-          perl -0777 -pe 's/resource::[0-9a-fA-F\-]+/resource::${{ secrets.BUSINESS_GROUP_ID }}/' -i.bak ${{ inputs.main_api_interface_path }}
-          cat ${{ inputs.main_api_interface_path }}
+          perl -0777 -pe 's/<AppenderRef ref="file" \/>/<!-- <AppenderRef ref="file" \/> -->/g' -i.bak src/main/resources/log4j2.xml
```

Comment:
I tried finding a way to do it in the `log4j2.xml` but haven't found any solution for our case. As I think we just need to enable this in LOCAL, and disable when not in LOCAL. But it seems log4j2 doesn't have any options for that.
Found [SystemPropertyArbiter](https://logging.apache.org/log4j/2.x/manual/configuration.html#arbiters), but didn't see the negate attribute, so the current implementation is ok to me